
## Longitudinal microbiome datasets

This project aims to integrate longitudinal microbiome cohorts to establish a comprehensive benchmark dataset. This benchmark will serve as a foundation for developing and evaluating our new approach to investigating microbiome dynamics.

The details of these datasets are listed below:


| Name  | Accession | Data type | Number of samples | Number of subjects | Description |
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
| Taur-2018 | PRJNA491657 |	16S (V4-V5)	 | 3736 |	778 | **Antibiotic treatment** during allo-HSCT disrupts gut microbiota diversity, increasing infection risk. A clinical trial shows that autologous fecal microbiota transplantation can restore microbial diversity and composition, highlighting its potential for post-antibiotic remediation. |
| Palleja-2018 |	PRJEB20800 |	Metagenome |	114 |	12 | A study on the gut microbiota of 12 healthy men revealed that a 4-day **antibiotic treatment** caused initial microbial shifts, but the microbiota largely recovered within 1.5 months. However, some species remained undetectable after 180 days, and resistance genes influenced recovery dynamics, indicating resilience despite antibiotic exposure. |
| Hagan-2019 |	PRJNA505336| 16S (V4) |	264 |	22 | A study on 22 healthy individuals found that **antibiotic treatment** before influenza vaccination impairs antibody response and enhances inflammatory signatures, with microbiome loss linked to immune system activation mechanisms. |
| Zhang-2022 |	PRJEB40730 |	Metagenome |	12276 |	887 | Study of gut microbiomes in genetically susceptible children reveals specific microorganisms associated with islet autoantibody seroconversion, highlighting functional genomic signatures linked to Type-1 diabetes risk. |
| Olm-2023-metagenome |	PRJEB49206 |	Metagenome |	39 |	39 | **Infant microbiomes** in nonindustrialized populations, such as the Hadza of Tanzania, show unique development trajectories with over 20% novel species, contrasting with industrialized infants who lack certain beneficial bacteria. Early life inheritance patterns highlight the need for broader microbiome studies beyond wealthy nations. |

## Key Features

**Dataset Collection:** Includes publicly available longitudinal microbiome cohorts studying both antibiotics intervention in healthy adults and microbiome development in young individuals.

**Standardization:** All datasets include host metadata and taxonomic profiles, either obtained from original publications or generated through our in-house pipeline, except the genome catalogue dataset.


## References

We acknowledge the following publications as critical resources for microbiome research, from which these datasets were sourced.

* [Taur, Ying, et al. "Reconstitution of the gut microbiota of antibiotic-treated patients by autologous fecal microbiota transplant." Science translational medicine 10.460 (2018): eaap9489.](https://doi.org/10.1126/scitranslmed.aap9489)

* [Palleja, Albert, et al. "Recovery of gut microbiota of healthy adults following antibiotic exposure." Nature microbiology 3.11 (2018): 1255-1265.](https://doi.org/10.1038/s41564-018-0257-9)

* [Hagan, Thomas, et al. "Antibiotics-driven gut microbiome perturbation alters immunity to vaccines in humans." Cell 178.6 (2019): 1313-1328.](https://doi.org/10.1016/j.cell.2019.08.010)

* [Zhang, Li, et al. "Islet autoantibody seroconversion in type-1 diabetes is associated with metagenome-assembled genomes in infant gut microbiomes." Nature communications 13.1 (2022): 3551.](https://doi.org/10.1038/s41467-022-31227-1)

* [Olm, Matthew R., et al. "Robust variation in infant gut microbiome assembly across a spectrum of lifestyles." Science 376.6598 (2022): 1220-1223.](https://doi.org/10.1126/science.abj297)